src/test/java/com/conveyal/gtfs/loader/JDBCTableWriterTest.java

```diff
@@ -69,6 +84,7 @@ public static void setUpClass() throws SQLException, IOException, InvalidNamespa
         testDBName = TestUtils.generateNewDB();
         String dbConnectionUrl = String.format("jdbc:postgresql://localhost/%s", testDBName);
         testDataSource = TestUtils.createTestDataSource(dbConnectionUrl);
+        connection = testDataSource.getConnection();
         LOG.info("creating feeds table because it isn't automatically generated unless you import a feed");
         Connection connection = testDataSource.getConnection();
```

Comment:
What is the difference between the two `connection` objects?

Reply:
None. Have reduced to using one.